.github/actions/generate-coverage/tests/test_scripts.py

```
@@ -2003,7 +2067,7 @@ def fake_run_cmd(*_: object, **__: object) -> None:
     monkeypatch.setattr(run_python_module, "run_cmd", fake_run_cmd)
 
     @contextlib.contextmanager
-    def fake_tmp_coveragepy_xml(out: Path) -> typ.Iterator[Path]:
+    def fake_tmp_coveragepy_xml(out: Path, _python: str) -> typ.Iterator[Path]:
```

Ruff (0.15.11)

[warning] Unused function argument: `out`

(ARG001)